README.rst

```diff
@@ -142,8 +142,8 @@ The repository consists mostly of externally hosted subrepositories:
 .. _zlib: https://github.com/madler/zlib
 .. |zliblic| replace:: zlib License
 .. _zliblic: https://github.com/madler/zlib/blob/master/README
-.. |zlibver| replace:: 1.2.12
-.. _zlibver: https://github.com/madler/zlib/releases/tag/v1.2.12
+.. |zlibver| replace:: 1.2.13
+.. _zlibver: https://github.com/madler/zlib/releases/tag/v1.2.13
```

Comment:
Do we still need zlib-CVE-2022-37434.patch in such case?

Reply:
Removed